src/containers/ViewProducts/index.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { View } from "react-native";
+import SearchBar from "../../components/SearchBar/SearchBar";
+import ViewProduct from "@components/Product"
+
+
+const ViewProducts = () => {
+    return (
+      <View>
+      <SearchBar></SearchBar>
+      <ViewProduct></ViewProduct>
+      </View>
+    );
+  };
+  
+  export default ViewProducts;
+  
```

Comment:
the name it doesn't makes senses!!!!!!
ListProductCard??????

Reply:
Done